docs/basics/ActionsManagers.md

```diff
@@ -0,0 +1,194 @@
+      return { isPinging: false };
+
+    default:
+      return state;
+  }
+};
+```
+
+Since filtering by a specific action type is so common, the actions stream redux-observable gives you is a special `ActionsObservable` which has a custom `ofType()` operator to reduce that boilerplate:
+
+```js
+const pingManager = action$ =>
+  action$.ofType('PING')
+    .delay(1000) // Asynchronously wait 1000ms then continue
+    .mapTo({ type: 'PONG' });
+```
+
+## A Real World Example
+
+Now that we have a general idea of what an Actions Manager looks like, let's continue with a more real-world example:
+
+```js
+// action creator
+const fetchUser = username => ({ type: FETCH_USER, payload: username });
+
+// process manager
+const fetchUserManager = action$ =>
+  action$.ofType(FETCH_USER)
+    .mergeMap(action =>
+      ajax.getJSON(`https://api.github.com/users/${action.payload}`)
+        .map(response => ({ type: FETCH_USER_FULFILLED, payload: response }))
+    );
+    
+// later...
+dispatch(fetchUser('torvalds'));
+```
+
+> We're using action creators (aka factories) like `fetchUser` instead of creating the action POJO directly. This is a Redux convention that is totally optional.
+
+We have a standard Redux action creator `fetchUser`, but also a corresponding Manager to orchestrate the actual AJAX call. When that AJAX call comes back, we map the response to a `FETCH_USER_FULFILLED` action.
+
+> Remember, Actions Managers take stream of **actions in** and return a stream of **actions out**. If the RxJS operators and behavior shown so far is unfamiliar to you, you'll definitely want to take some time to [dive deeper into RxJS](http://reactivex.io/rxjs/) before proceeding.
+
+You can then update your Store's state in response to that `FETCH_USER_FULFILLED` action:
+
+```js
+const users = (state = {}, action) => {
+  switch (action.type) {
+    case 'FETCH_USER_FULFILLED':
+      return {
+        ...state,
+        // `login` is the username
+        [action.payload.login]: action.payload
+      };
+
+    default:
+      return state;
+  }
+};
+```
+
+## Accessing the Store's State
+
+Your Actions Managers receive a second argument, the Redux store itself.
+
+```js
+function (action$: Observable<Action>, store: Store): Observable<Action>;
+```
+
+With this, you can call `store.getState()` to synchronously access the current state:
+
+```js
+const INCREMENT = 'INCREMENT';
+const INCREMENT_IF_ODD = 'INCREMENT_IF_ODD';
+
+const incrementIfOdd = () => ({ type: INCREMENT_IF_ODD });
+const increment = () => ({ type: INCREMENT });
+
+const incrementIfOddManager = (action$, store) => (
+  action$.ofType(INCREMENT_IF_ODD)
+    .filter(() => store.getState().counter % 2 === 0)
+    .map(increment);
+);
+
+// later...
+dispatch(incrementIfOdd());
+```
+
+Remember, `store.getState()` is just an imperative, synchronous API. You cannot treat it as a stream as-is. While it's not a common pattern for Managers to need, Redux does support a stream of state changes:
```

Comment:
That would just be sugar for `Observable.from(store)`. I'm indifferent either way. We can either provide guidance to do so, or we can just provide the argument. The only thing I'd want to see before adding it to the API would be some half-reasonable argument for the demand to use it. Doesn't even need to be a good argument, since it's a trivial add.